Makefile

```diff
@@ -1,7 +1,7 @@
 .PHONY run test:
 
 run:
-	cargo run
+	cargo run $(filename)
```

Comment:
Pretty cool! I think you could write a variable with a default value in case the user doesn't pass a filename. Like so:

```make

FILENAME.   ?= ./examples/2048.obj
un:
	cargo run
	cargo run $(FILENAME)
```

Plus, I think makefile variable are usually capitalized. I might be making this up.